pygerduty/v2.py

```diff
@@ -57,6 +57,7 @@ def __init__(self, pagerduty, base_container=None):
 
         self.pagerduty = pagerduty
         self.base_container = base_container
+        self.type = None
```

Comment:
I think I added it and was going to use it but ended up not using it. I will remove.